app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -28,6 +28,10 @@ class Contact extends React.Component {
           value: this.state.current.patient.preferred_contact_method,
         },
       });
+    }
+    // There are two instances when a user might already have an email that we'd want to prefill the confirm email field
+    // One is editing an existing monitoree. The other is when enrolling a close contact
+    if (this.state.current.patient.email) {
```

Comment:
> Now checks for the existence of an email address to decide whether to prefill the confirm_email field. Before, it was only when editing an existing monitoree, but enrolling a Close Contact should prefill their email to the confirm_email as well.

I see that you added the check (which is a good call), but it doesn't look like this was only happening when editing a monitoree - seems like it was always happening. Am I missing something?

Reply:
I missed the move out of the `isEditMode` block in the diff. Resolving.